scrapy/czlscrape/__init__.py

```diff
@@ -1,7 +1,11 @@
 import os
+import time
+import logging
 
 if 'SENTRY_DSN' in os.environ:
     import logging
     from raven.handlers.logging import SentryHandler
     from raven.conf import setup_logging
     setup_logging(SentryHandler(os.environ['SENTRY_DSN'], level=logging.WARN))
+
+logging.Formatter.converter = time.gmtime
```

Comment:
@mgax gmtime makes it GMT. UTC..ish. Buut, a quick test shows timezone information is in fact lost / never set:
```python
In [1]: import time

In [2]: time.strftime('%Y-%m-%dT%H:%M:%SZ', time.gmtime())
Out[2]: '2017-04-06T09:57:34Z'

In [3]: time.strftime('%Y-%m-%dT%H:%M:%SZ', time.localtime())
Out[3]: '2017-04-06T12:57:37Z'
```
@margelatu I didn't think about prod being someplace else. Though I still think we'd be much better off with times that are relevant to us. (and I just figured out this discussion is relevant for scraped data too, since that stuff is timezone-unaware initially it might get converted in unexpected ways).
I'd even propose the machine timezone be set to Europe/Bucharest regardless of where it's located (nothing should go wrong.... right?). Or at least convert the local timezone to something configurable in settings.